src/deluge/hid/hid_sysex.cpp

```diff
@@ -28,10 +37,61 @@ void HIDSysex::requestOLEDDisplay(MIDIDevice* device, uint8_t* data, int32_t len
 	if (data[4] == 0 or data[4] == 1) {
 		sendOLEDData(device, (data[4] == 1));
 	}
+	else if (data[4] == 2 || data[4] == 3) {
```

Comment:
Nit: Currently the protocol parsing looks magic number heavy. Maybe add some enums for clarity?